### ISSUE TITLE
LSP: imports from ts -> civet are always reported as erroneous

### ISSUE THREAD

**Issue by lorefnon** [DanielXMoore/Civet#60]
![image](https://user-images.githubusercontent.com/1449492/208239747-b761e991-e0bd-43aa-9632-43b1a8c8d8bb.png)

**Comment by STRd6** [comment]
Is the `@builtin` TypeScript and JavaScript Language Features extension disabled for your workspace?

**Comment by lorefnon** [comment]
I tried disabling that but that doesn't seem to help.

**Comment by STRd6** [comment]
Try LSP version 0.3.2 and let me know if that fixes it. I hadn't registered the extension for the `typescriptreact` language identifier.

**Comment by lorefnon** [comment]
@STRd6 I had updated that but surprisingly it continued to happen. Later upon debugging I found that the diagnostics that were sent by our lsp client did not contain the error. 

Once it was clear that some other ext was responsible for this error, I located the origin as volar plugin for vue, which does something similar to ours and gets activated on files that have nothing to do with vue. After disabling that too it went away.
